default.yaml

```diff
@@ -38,6 +38,7 @@ test_accounts:
   code: 321321
 - phone: "+16505554322" # user1
   code: 321432
+  phoneMetadata_carrier_type: "mobile" # TODO: verify mobile is a valid value
```

Comment:
Please change to camel case. 

in other configs we use camel case. Here we are using both (camel and snake case)